templates/addons/aurora/mysql-cf.yml

```diff
@@ -66,77 +58,81 @@ Resources:
               StringEquals:
                 'kms:CallerAccount': !Ref 'AWS::AccountId'
                 'kms:ViaService': !Sub 'rds.${AWS::Region}.amazonaws.com'
-
   AuroraKMSCMKAlias:
     Type: 'AWS::KMS::Alias'
     DeletionPolicy: Retain
-    DependsOn: ['AuroraDBCluster']
+    DependsOn: ['{{logicalIDSafe .Name}}']
     Properties:
-      AliasName: !Sub 'alias/${AuroraDBCluster}'
+      AliasName: !Sub 'alias/${ {{- logicalIDSafe .Name -}} }'
       TargetKeyId: !Ref AuroraKMSCMK
-
   DBSubnetGroup:
     Type: 'AWS::RDS::DBSubnetGroup'
     Properties:
-      DBSubnetGroupDescription: Group of Copilot private subnets for Aurora cluster. 
+      DBSubnetGroupDescription: Group of Copilot private subnets for Aurora cluster.
       SubnetIds:
         !Split [',', { 'Fn::ImportValue': !Sub '${App}-${Env}-PrivateSubnets' }]
-
   DBClusterSecurityGroup:
     Type: AWS::EC2::SecurityGroup
     Properties:
-      GroupDescription: The Security Group for the database cluster. 
+      GroupDescription: The Security Group for the database cluster.
       SecurityGroupIngress:
-        - ToPort: !Ref DBPort
-          FromPort: !Ref DBPort
+        - ToPort: 3306
+          FromPort: 3306
           IpProtocol: tcp
           Description: From Environment Security Group
-          SourceSecurityGroupId: 
-            Fn::ImportValue: 
+          SourceSecurityGroupId:
+            Fn::ImportValue:
               !Sub '${App}-${Env}-EnvironmentSecurityGroup'
-      VpcId: 
-        Fn::ImportValue: 
+      VpcId:
+        Fn::ImportValue:
           !Sub '${App}-${Env}-VpcId'
       SecurityGroupEgress:
         - IpProtocol: '-1'
           Description: Allow all outbound traffic by default
           CidrIp: 0.0.0.0/0
-
   AuroraSecret:
     Type: AWS::SecretsManager::Secret
     Properties:
-      Name: !Sub '/${App}/${Env}/${Name}/${DBTableName}'
-      Description: !Sub Aurora PostgreSQL main user secret for ${AWS::StackName}
+      Name: !Sub '/${App}/${Env}/${Name}'
+      Description: !Sub Aurora main user secret for ${AWS::StackName}
       GenerateSecretString:
-        SecretStringTemplate: '{"username": "postgres"}'
+        SecretStringTemplate: '{"username": "admin"}'
         GenerateStringKey: "password"
         ExcludePunctuation: true
         IncludeSpace: false
         PasswordLength: 16
-  
+  {{- if .ParameterGroup}}
+  # DBClusterParameterGroup:
+  #   Type: 'AWS::RDS::DBClusterParameterGroup'
+  #   Properties:
+  #     Description: !Ref 'AWS::StackName'
+  #     Family: 'aurora-mysql5.7'
+  #     Parameters:
+  #       character_set_client: 'utf8'
```

Comment:
Why do we comment this out?

Reply:
Btw this is in #2075. I will rebase this PR once #2075 is merged.

I wanted to render this part of the template even if the customers are bringing their own parameter group. Since the add-on template we render is more of a guide/starting point and customers can customize them all they want, I think this might be a useful piece of "guidance" that we can provide. 

So if they decide to use our default parameter group, they can just uncomment this chunk of code and modify the parametergroup property for the db cluster resource.